bfscripts.spec.rpkg

```diff
@@ -58,8 +58,6 @@ install mlx-mkbfb %{installdir}
 mkdir -p %{buildroot}%{_unitdir}/
 install bfvcheck.service %{buildroot}%{_unitdir}/
 
-ln -s /opt/mlnx %{buildroot}/opt/mellanox
-
```

Comment:
Any plan how to sync-up with Vlad? Are we going to create a symbolic link, or coordinate the changes with Vlad's scripts?

Reply:
The plan is to create a symbolic link in /opt/mlnx/scripts to each script during the install process, and warn users that we'll be deprecating the /opt locations.